src/sipnet/sipnet.c

```diff
@@ -986,7 +986,7 @@ void calcRootResp(double *rootResp, double respQ10, double baseRate,
 // This is in addition to [1], source not yet determined (controlled by
 // ctx.growthResp)
 void vegResp2(double *folResp, double *woodResp, double *growthResp,
-              double baseFolResp) {
+              double baseFolResp, double gpp) {
```

Comment:
```suggestion
              double baseFolResp) {
```
I think this was an incorrect merge, this arg should have gone away